src/frontend/screens/ObservationDetails/Question.tsx

```diff
@@ -8,19 +7,18 @@ import TextArea from "./TextArea";
 import type { Field } from "../../context/ConfigContext";
 
 export type QuestionProps = {
-  field: Field,
-  value: any,
-  onChange: (value: any) => any,
+  field: Field;
+  value: any;
+  onChange: (value: any) => any;
+  autoFocus?: boolean;
 };
 
-const Question = (props: QuestionProps) => {
-  // This is needed for Flow to understand the type refinement
-  const { field, ...other } = props;
+const Question = ({ field, autoFocus = false, ...other }: QuestionProps) => {
```

Comment:
Minor niggle, not sure this makes sense as a prop for this component, given that it has no meaning if the field is a select_one or select_multiple. Is there a need to expose it? I think we want all questions that are text fields to autofocus.

Reply:
I agree that not necessary to expose it for the select_one and the select_multiple but I did this for simplicity. So the text area take `QuestionProps`. If we want to expose it to ONLY the text field, I would need to create:
> interface TextFieldProp{
> questionProps:QuestionProps,
> autofocus: boolean
>}

The way I have it right now, by making autofocus optional seems a lot simpler to me.

Reply:
I think given Gregor's suggestion, you'd update https://github.com/digidem/mapeo-mobile/blob/9051b94135f8551f422ee2824fb66da82d638f9c/src/frontend/screens/ObservationDetails/TextArea.tsx#L18 so that it's 

```jsx
  <>
    <QuestionLabel field={field} />
    <TextInput
      value={value}
      onChangeText={onChange}
      style={styles.textInput}
      underlineColorAndroid="transparent"
      multiline
      scrollEnabled={false}
      textContentType="none"
      autoFocus // <-- Adjusted line
    />
  </>
```

which wouldn't require you to update the `QuestionProps` in `Question.tsx`